src/SDKs/ApiManagement/Management.ApiManagement/Microsoft.Azure.Management.ApiManagement.csproj

```diff
@@ -8,7 +8,7 @@
     <Description>Provides ApiManagement management capabilities for Microsoft Azure.</Description>
     <AssemblyTitle>Microsoft Azure API Management Management</AssemblyTitle>
     <AssemblyName>Microsoft.Azure.Management.ApiManagement</AssemblyName>
-    <Version>4.0.6-preview</Version>
+    <Version>5.0.0-preview</Version>
```

Comment:
We generally recommend starting a stable version with the next major version. In this case, it would be good to start 5.0.0`. However if you wish to keep this preview, please start from `4.8.0-preview` and bump to `5.0.0` when ready